### PR TITLE
Fb login

### PR DESCRIPTION
**Pull request by umangahuja1**
Review and tell me the chromedriver installation for other os I have just included for linux.

### CODE REVIEW

**Comment by umangahuja1** [comment]
These test are passing on my pc both coala --ci and unittest too. What's the matter.

**Comment by umangahuja1** [comment]
Done.

**Comment by umangahuja1** [comment]
Is there any problem with this PR.

**Comment by appi147** [comment]
@umangahuja1 On gitter

File: jarviscli/packages/fb.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                f.write(usr)
+                f.write('\n')
+                f.write(pwd)
+    return usr, pwd
+
+
+def fb_login(self):
+    usr, pwd = get_details()
+
+    driver = webdriver.Chrome()
```
Comment (appi147) [inline jarviscli/packages/fb.py]: driver = webdriver.Chrome("/usr/lib/chromium-browser/chromedriver")

Comment (appi147) [inline jarviscli/packages/fb.py]: Test this once and push

Comment (umangahuja1) [inline jarviscli/packages/fb.py]: the thing is I have chromedriver installed so my pc pass this script anyways. How can I test like someone else is testing.

**Pull request closed without merging**